gravatar/openapi/api-gravatar.json

```diff
@@ -23,6 +23,14 @@
     {
       "name": "avatars",
       "description": "Operations about user avatars"
+    },
+    {
+      "name": "qr-code",
+      "description": "Operations about QR codes"
+    },
+    {
+      "name": "experimental",
+      "description": "Experimental operations that might be subject to change. Use with caution."
     }
```

Comment:
I think we don't have any of those in the SDK, right?

Reply:
🤦 I removed other methods we currently don't support, but forgot to remove that one. Updated!

Thanks!